### DISCUSSION TITLE
Version 3.3 forces X and Y to be 0 on child svg

### DISCUSSION THREAD

**Discussion by rparussel** [vvvv/SVG#948] [General]
Previously, when using version 3.2.3, I made use of a trick to create an svg that would be resized with a new aspect ratio to fill it's container, which contained text that retained their aspect ratio. The text was placed inside child svg's within the parent svg. In order to do this it was necessary to set the X and Y values on the child svg's to position the text. Here's a snippet of the html that it generated ...

![image](https://user-images.githubusercontent.com/64085407/151352820-76f56505-8072-4351-bb05-e13333f80fa8.png)

However, I've since upgraded to version 3.4 and I notice that all X and Y values on svg's are forced to be 0. Looking at the release notes this seems to have been done as a fix in version 3.3 ...

* ignore `x` and `y` attributes in outermost svg as per standard (see #886 ([https://github.com/svg-net/SVG/issues/886]))

In my case these child svg's, which are not outmost svg's are also affected. Looking at the source for SvgDocument, the cause seems clear ...
![image](https://user-images.githubusercontent.com/64085407/151354938-0073df4a-4c97-439b-b17b-8a983491a15c.png)

Would it be possible to include a fix in the next release to only set X and Y to zero on the outermost svg rather than on the child svg's as well?

**Comment by H1Gdev** [comment]
I think outermost \<svg\> is `SvgDocument` and inner \<svg\>s is `SvgFragment`.

outermost part is fixed to 0 according to specifications of w3c SVG.

**Reply by rparussel** [reply]
Thanks, @H1Gdev . Using SvgFragment resolves the issue.
